### DISCUSSION TITLE
Please remove the 'beta' version as the default project view.

### DISCUSSION THREAD

**Discussion by TJKoury** [community/community#11193] [Projects and Issues]
Very frustrating to send a team member to the projects page, then have to direct them to the actual projects from the drop-down because it says "0 projects".

**Comment by Ncookiez** [comment]
This is particularly annoying since the beta organization projects page remains even when organization-wide projects are disabled. Pretty much a bug at this point.

**Comment by bobbrow** [comment]
Agree.  I'd like to remove beta projects by default or merge them in with "normal" projects so I don't have to click through another level every time I go to the Projects tab.

**Comment by xelderon** [comment]
Is this even being considered? It is incredibly annoying.

**Comment by Malvoz** [comment]
More requests for this:
- https://github.com/github-community/community/discussions/13386
- https://github.com/github-community/community/discussions/19870
